### ISSUE TITLE
gengraphs enhancement: switch to choose output format (.svg, .png)

### ISSUE THREAD

**Issue by mar-mor** [Mastercard/p11perftest#3]
Currently, running gengraphs.py outputs graphs in png and svg formats. 

Typically, a user only wants png. 

It would be good to add a switch that determines output file:
gengraphs.py <file.xlsx> -o {svg,png}

The default (if no switch given) should produce both png and svg

**Comment by keldonin** [comment]
I would suggest another switch than `-o`, which typically implies an output file. is `-t`available?

**Comment by mar-mor** [comment]
> I would suggest another switch than `-o`, which typically implies an output file. is `-t`available?

`-t` is used for table name (sheet name in excel terminology) currently. 

could replace `-t --table` with `-s --sheet`, to make `t` available for output type

**Comment by keldonin** [comment]
what flags are already used?

**Comment by mar-mor** [comment]
```optional arguments:
  -h, --help            show this help message and exit
  -t TABLE, --table TABLE
                        Table name
  -e, --error           Remove error regions from plot
  -c FILE, --comparison FILE
                        Compare two datasets. Provide the path to a second Excel spreadsheet.
  -l LABELS LABELS, --labels LABELS LABELS
                        Dataset labels (defaults to "data set 1" and "data set 2")
```
There's also `--reglines` for throughput vs size

**Comment by keldonin** [comment]
what about `-f`and `--format` ?
Also, I would suggest to change `--error` to `--no-error-region` or something similar.

**Comment by mar-mor** [comment]
addressed by commit 8062a63
